.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please include a summary of the change and which (if so) issue is fixed.
+
+Fixes #(ISSUE_NUMBER_HERE)
+
+## Checklist
+
+Please delete options that are not relevant.
+
```

Comment:
I think every item on this checklist should be run, right?

Reply:
Fixed! :)